### PR TITLE
add a section about the compiler team

### LINKED ISSUE #25: include a section on how the compiler team functions
We should talk about how the compiler team itself functions:

- [x] Chatting typically in `#rustc` on IRC
- [x] Open-to-public triage meetings in `#rustc` on IRC at 10am Boston time on Thursday
  - Careful around US daylight savings time :/
- [x] How to find open issues
- [x] Mentoring and how to ask for it (#64)

### PR DESCRIPTION
**Pull request by nikomatsakis**
Fixes #25

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - [About this guide](./about-this-guide.md)
 - [How to build the compiler and run what you built](./how-to-build-and-run.md)
 - [Coding conventions](./conventions.md)
+- [About the compiler team](./compiler-team.md)
```
Comment (mark-i-m) [inline src/SUMMARY.md]: I would perhaps make this the first chapter after "about this guide"

File: src/compiler-team.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- **Check I-nominated issues:** These are issues where feedback from
+  the team is desired.
+- **Check for beta nominations:** These are nominations of things to
+  backport to beta.
+
+The meeting currently takes place on Thursdays at 10am Boston time
+(UTC-4 typically, but daylight savings time sometimes makes things
+complicated).
+
+The meeting is held over a "chat medium" -- it used to be IRC, but we
```
Comment (mark-i-m) [inline src/compiler-team.md]: Sorry, the same silly nit: `--` -> em-dash

File: src/compiler-team.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+for you.
+
+[hi5]: https://github.com/rust-highfive
+
+Getting on the high-five list is much appreciated as it lowers the
+review burden for all of us! However, if you don't have time to give
+people timely feedback on their PRs, it may be better that you don't
+get on the list.
+
+### Team membership
```
Comment (mark-i-m) [inline src/compiler-team.md]: Perhaps rename to `Full Team Membership`? There is already a section entitled "Team membership".

**Comment by mark-i-m** [comment]
Thanks! If you have time, could you also add some of the other items you mention in #25? Then we could close that issue.

File: src/compiler-team.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+- **Check for beta nominations:** These are nominations of things to
+  backport to beta.
+
+The meeting currently takes place on Thursdays at 10am Boston time
+(UTC-4 typically, but daylight savings time sometimes makes things
+complicated).
+
+The meeting is held over a "chat medium" -- it used to be IRC, but we
+are currently in the process of evaluating other alternatives. Check
+[the etherpad] to find the current home (and see
```
Comment (wesleywiser) [inline src/compiler-team.md]: This link is `[the etherpad]` but the source below is `[etherpad]` so the link doesn't render.

**Comment by mark-i-m** [comment]
Thanks!

**Pull request merged**